corehq/apps/change_feed/README.md

```diff
@@ -31,3 +31,5 @@ cd /usr/local/opt/kafka/libexec
 
 If you use the default configuration you should not have to do anything.
 If you are running kafka in a VM, on another machine, or on a nonstandard port, you will need to override `KAFKA_URL` in your `localsettings.py`.
+
+To more easily manage Kafka's settings, try installing [kafkat](https://github.com/airbnb/kafkat)
```

Comment:
You mean the `kafka-administer` command? It should, as long as you can connect to it via http.  It uses the same kafka client we use in the [changes feed](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/change_feed/connection.py#L7-L9).

I assume kafkat will work locally too, but I haven't tried it.  You have to make an `/etc/kafkatcfg`.  Check [ansible](https://github.com/dimagi/commcarehq-ansible/blob/master/ansible/roles/kafka/templates/kafkatcfg.j2) for specifics.